docs-next/pages/apis/config.mdx

```diff
@@ -7,20 +7,36 @@ The `keystone-next` CLI expects to find a module named `keystone.ts` with a defa
 ```typescript filename=keystone.ts
 import { config } from '@keystone-next/keystone/schema';
 
-export default config({ /* ... */ });
+export default config({
```

Comment:
I know I'm going to lose this battle at some point, but could you review this PR for all the places our dots get spread over three lines and put them back on one? 🙏

Reply:
will diddly do